src/Sensors.cpp

```diff
@@ -366,6 +365,18 @@ void Sensors::setTempOffset(float offset) {
   setsen5xTempOffset(toffset);
 }
 
+/**
+ * @brief Get temperature offset for Sensirion sensors (from internal sensor in SCD4x and SCD30)
+ * @return float with the temperature offset.
+ * Positive value for offset to be subtracetd to the temperature.
+ */
+float Sensors::getTempOffset() {
+  float toffset;
+  toffset = getSCD30TempOffset();
```

Comment:
Maybe we need also here a validation in each call, if the sensor is registered, because the last one call could override the first one with 0.0